packages/css/src/components/Popover.css

```diff
@@ -0,0 +1,30 @@
+.Popover {
+  --spacing: var(--space-s);
+
+  opacity: 0;
+  transition: opacity 0.1s ease-out;
+  padding: var(--space-s) var(--space-m);
+  color: var(--color-white);
+  background-color: rgba(25, 1, 52, 0.8);
+  border-radius: 0.25rem /* 4px */;
+}
+
+.Popover-wrapper:hover + .Popover {
+  opacity: 1;
+}
+
+.Popover--top {
+  margin-bottom: var(--spacing);
```

Comment:
It defines the space between the content and the popover content

Reply:
Yeah, I'm just referring to the variable name `--spacing`
Shouldn't we have like `--spacing-m`, `--spacing-l` etc?

Reply:
`--spacing-m` and `--spacing-l` are global variables when `--spacing` is declared in `.Popover`

I'll rename it to `--spaceAround` to avoid confusion